shared/teams/main/index.js

```diff
@@ -30,6 +30,16 @@ const Teams = (props: Props) => (
         )}
         <TeamList {...props} />
         <BetaNote {...props} />
+        {isMobile && (
```

Comment:
I'm guessing you do this for mobile because the progress indicator won't fit in the header? May be worth a comment

Reply:
yup that's why, added a comment!